_drafts/SC3-3-2-form-field-has-name.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Form field has accessible name
+description: |
+  Each form field element has an accessible name
+
+success_criterion:
+- 3.3.2 # Labels or Instructions
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rules applies to:
+- any element that is or [exposed to assistive technologies](#exposed-to-assistive-technologies),
+- that has a [semantic role](#semantic-role) that inherits from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and 
+- does not have a required context role that itself inherits from one of those roles.
```

Comment:
You could link required context role also to its ARIA definition: https://www.w3.org/TR/wai-aria/#scope